packages/server-wallet/src/message-service/test-message-service.ts

```diff
@@ -0,0 +1,26 @@
+import {Message} from '@statechannels/client-api-schema';
+import _ from 'lodash';
+
+import {MessageHandler, MessageServiceInterface} from './types';
+
+export class TestMessageService implements MessageServiceInterface {
+  protected constructor(private _receive: MessageHandler) {}
+
+  static async createTestMessageService(
+    incomingMessageHandler: MessageHandler
+  ): Promise<MessageServiceInterface> {
+    const service = new TestMessageService(incomingMessageHandler);
+    return service;
+  }
+
+  async send(messages: Message[]): Promise<void> {
+    for (const message of messages) {
+      await this._receive(message.recipient, message.data, this);
```

Comment:
```suggestion
      await this._receive(message, this);
```

Refactor (to match other suggestion)

Reply:
Resolved in 0accce4